cli/compose/convert/service.go

```diff
@@ -217,13 +217,17 @@ func convertServiceSecrets(
 		if gid == "" {
 			gid = "0"
 		}
+		mode := secret.Mode
+		if mode == 0 {
```

Comment:
@mstanleyjones @AkihiroSuda yeah.. didn't thought of that, but yeah should probably be a pointer instead.